gen/DSE.Open.Values.Generators/ValueTypesGenerator.cs

```diff
@@ -186,17 +187,21 @@ or TypeNames.ComparableValueAttributeFullName
                         continue;
                     }
 
-                    var nameEquals = attributeArgs.Arguments.Where(a => a.NameEquals is not null).ToArray();
+                    var attributeArgNames = attributeArgs.Arguments.Where(a => a.NameEquals is not null).ToArray();
 
-                    if (nameEquals.Length == 0)
+                    if (attributeArgNames.Length == 0)
                     {
                         continue;
                     }
 
-                    var maxSerializedCharLengthSyntax = nameEquals.FirstOrDefault(a =>
+                    var maxSerializedCharLengthSyntax = attributeArgNames.FirstOrDefault(a =>
                         a.NameEquals is not null
                         && a.NameEquals.Name.Identifier.ValueText == "MaxSerializedCharLength");
 
+                    var allowEmptySyntax = attributeArgNames.FirstOrDefault(a =>
+                        a.NameEquals is not null
+                        && a.NameEquals.Name.Identifier.ValueText == "AllowEmpty");
```

Comment:
Should this be "AllowDefault" - empty makes sense for a value with a string or collection of values underpinning it, but not for a primitive value like int

Reply:
Done